src/app/pages/login/login.page.ts

```diff
@@ -47,16 +47,6 @@ export class LoginPage implements OnInit {
     }
     
     login(){
-        this.loginService.getUserInfo()
```

Comment:
@LucDelp on a un probleme : si un utilisateur se connecte et se deconnecte et qu'ensuite il veux se connecter a un autre compte, il pourra pas non?

Reply:
si puisque à la déconnexion, on supprime toutes les informations perso de l utilisateur, donc la value retourner sera null / et ca va trigger le "this.presentAlert()", qui montre le formulaire de connexion. Je le fais deja quand je switch entre les 2 comptes de tests que j ai créé